ted2zim/scraper.py

```diff
@@ -291,6 +291,20 @@ def generate_subtitle_list(self, video_id, langs, curr_lang):
             ]
         return build_subtitle_pages(video_id, subtitles)
 
+    def generate_urls_for_other_languages(self, url):
+        """Given a TED video URL, this generates possible URLs for other languages requested"""
```

Comment:
Use concise wording for docstring, explaining the return value or the action done. Context is already defined by method signature. ex:

``` py
""" Possible URLs for other requested languages based on a video url """
```

Reply:
Changed this